.travis.yml

```diff
@@ -24,9 +24,8 @@ matrix:
           sudo: required
           dist: trusty
           group: edge
-        - php: 5.5
-        - php: 5.6
-        - php: 7.0
+        - php: 7.1.0
```

Comment:
Unless I misunderstood what you meant, the line your asking for doesn't exist on the SF3 lines. Thus we should not add it since it has never been required.